src/styles/card.scss

```
@@ -70,6 +78,8 @@
     text-transform: uppercase;
     text-decoration: none;
     color: #fff;
+
+    transition: 1s;
     &:hover {
       background-color: #fff;
```

```suggestion
    transition: background-color 1s;
    &:hover {
      background-color: #fff;
```
В свойстве **transition** надо прописывать свойства к которым оно применяется. Иначе при добавлении какого-то свойства в &:hover **transition** применится и на него, а вдруг это тебе не нужно было:)